pontoon/contributors/templates/contributors/includes/timeline.html

```diff
@@ -1,22 +1,27 @@
-<h3>{{ contribution_timeline.title }}</h3>
+<h3>Contribution activity</h3>
 
-{% for title, values in contribution_timeline.contributions.items() %}
+{% for month, data in contribution_timeline.contributions.items() %}
+{{month}}
 <div class="contribution-group">
-    <h4 class="heading">
-        <span class="icon far {{ values.type }}"></span>
-        {{ title }}
-    </h4>
-    <ul class="localizations">
-        {% for _, localization in values.data.items() %}
-        <li>
-            <a href="{{ localization.url }}">
-                <span class="stress">{{ localization.project.name }}</span> &middot; {{
-                localization.locale.name }} <span class="stress">{{ localization.locale.code }}</span>
-                <span class="contribution-count">{{ ", ".join(localization.actions) }}</span>
-            </a>
-        </li>
-        {% endfor %}
-    </ul>
+    {% for _, info in data.items() %}
+        <div class="contribution-type">
+            <h4 class="heading">
+                <span class="icon far {{ info.type }}"></span>
+                {{ info.title }}
+            </h4>
+            <ul class="localizations">
+                {% for key, localization in info.data.items() %}
```

Comment:
Since we don't use `key`, let's do this:

```suggestion
                {% for _, localization in info.data.items() %}
```